functions/src/index.ts

```diff
@@ -1,14 +1,15 @@
-'use strict';
-
 import * as functions from 'firebase-functions'
-import * as express from 'express'
-import * as admin from 'firebase-admin'
+import express from 'express'
+import admin from 'firebase-admin'
 import RedirectHandler from "./RedirectHandler";
 import Api from "./api/Api";
-import * as cookieParser from "cookie-parser";
+import cookieParser from "cookie-parser";
+import dotenv from "dotenv"
+
+dotenv.config()
```

Comment:
Did you have any `.env` config required for production? Pls, put here any sample file and add installation instructions to `README.md` file.

Reply:
Maybe it's whole unnecessary, because Firebase is using `dotfile` internally: https://firebase.google.com/docs/functions/config-env#env-variables